books-call-number/nodes/js/buildQuery.js

```diff
@@ -2,10 +2,12 @@ if (logLevel === 'DEBUG') {
   print('\nlogLevel = ' + logLevel + '\n');
   print('\ncall number start range = ' + startRange + '\n');
   print('\ncall number end range = ' + endRange + '\n');
+  print('\nlocationName = ' + locationName + '\n');
 }
-
 var where = 'TRUE';
```

Comment:
```suggestion

var where = 'TRUE';
```
Add back missing new line.